systems/plants/RigidBodyManipulator.cpp

```diff
@@ -533,6 +534,12 @@ void RigidBodyManipulator::compile(void)
   cached_q.resize(num_positions);
   cached_v.resize(num_velocities);
 
+  cached_q_new.resize(num_positions);
+  cached_v_new.resize(num_velocities);
+
+  cached_q_old.resize(num_positions);
+  cached_v_old.resize(num_velocities);
```

Comment:
yuck.  can you put this comment into the code (so we remember when we go back and clean up?)